cmd/bridge/config.go

```diff
@@ -70,8 +70,9 @@ func SetFlagsFromConfig(fs *flag.FlagSet, filename string) (err error) {
 		return err
 	}
 
-	if config.APIVersion != "console.openshift.io/v1beta1" || config.Kind != "ConsoleConfig" {
-		return fmt.Errorf("unsupported version (apiVersion: %s, kind: %s), only console.openshift.io/v1beta1 ConsoleConfig is supported", config.APIVersion, config.Kind)
+	if !((config.APIVersion == "console.openshift.io/v1beta1" && config.Kind == "ConsoleConfig") ||
```

Comment:
I believe we want to stick with calling it ConsoleConfig both times:
```
console.openshift.io/v1beta1    ConsoleConfig
config.openshift.io/v1    ConsoleConfig // instead of Console
```
At this point the `console` has never seen `Kind = Console` (but the operator has seen it from `openshift-config-managed`), so we won't break anything.